pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -288,9 +291,10 @@ func cleanup(t *testing.T, m Manager, p *Stub) {
 
 func TestUpdateCapacityAllocatable(t *testing.T) {
 	socketDir, socketName, _, err := tmpSocketDir()
+	topologyStore := topologymanager.NewFakeManager()
 	require.NoError(t, err)
 	defer os.RemoveAll(socketDir)
-	testManager, err := newManagerImpl(socketName)
+	testManager, err := newManagerImpl(socketName, topologyStore)
 	as := assert.New(t)
 	as.NotNil(testManager)
 	as.Nil(err)
```

Comment:
This patch also needs to include the following change in order for the original `devicemanager` tests to pass:
```
@@ -602,6 +606,7 @@ func getTestManager(tmpDir string, activePods ActivePodsFunc, testRes []TestReso
        allocatedDevices:  make(map[string]sets.String),
        endpoints:         make(map[string]endpointInfo),
        podDevices:        make(podDevices),
+       topologyAffinityStore: topologymanager.NewFakeManager(),
        activePods:        activePods,
        sourcesReady:      &sourcesReadyStub{},
        checkpointManager: ckm,
```